mitodl/bootcamp-ecommerce
# Added NovoEd integration for adding/removing enrollments

#### What are the relevant tickets?
Closes #999

#### What's this PR do?
- Enrolls a user in a NovoEd course when they enroll on the bootcamp side and that bootcamp run is properly configured
- Unenrolls the user from NovoEd when they are refunded
- Adds a management command to backfill NovoEd enrollments

#### How should this be manually tested?
First, ask me for the API key and secret

Then:
- Add the feature flag (`FEATURE_NOVOED_INTEGRATION=True`)
- Add a `novoed_course_stub` value for some bootcamp run (this will be a specific value that I'll send over)
- Enroll in that bootcamp run

After the order processes, you should see that user in NovoEd user management

#### Any background context you want to provide?
NovoEd is another courseware provider that at least some bootcamps will be using to deliver their content
